spec/deaf_grandma_spec.rb

```diff
@@ -10,9 +10,12 @@
   
   it "says 'NOT SINCE 1964!' when we yell" do
     #implement your test here
+    expect(script.speak("HI GRANDMA")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    expect(script.speak("BYE")).to eq "NOT SINCE 1964!"
+    expect(script.speak("BYE")).to eq "NOT SINCE 1964!"
+    # Need some way to Mock exit.
```

Comment:
You may want to just test for grandma's final response here and not actually test for the termination of the script.